src/main/java/create_task_use_case/CreateTaskOutputData.java

```diff
@@ -0,0 +1,47 @@
+package create_task_use_case;
+
+/**
+ * Task Creation Output Data Class. Instances of this class are returned upon the
+ * execution of the application logic for this use case. The errorMessage and
+ * isSuccessfullyModified instance attributes, in particular, represent whether the use
+ * case logic was successfully executed.
+ * @author Fardin Faruk
+ */
+public class CreateTaskOutputData {
+
+    private final String title;
+    protected boolean isSuccessfullyCreated;
+    private final int dayIndex;
+    private String errorMessage;
+
+
+    public CreateTaskOutputData(String title, int dayIndex, String errorMessage){
```

Comment:
Should probably remove the errorMessage parameter since empty string is always passed in and set it to the empty string in the variable declaration.

Reply:
Done.